tests/software/test_conda.py

```diff
@@ -320,7 +320,7 @@ def test_from_file_duplicate_subtype(self, tmp_path):
                 channels:
                 - defaults
                 dependencies:
-                - python=3.7.11
+                - python=10.11
```

Comment:
looks like a typo? should be 3.10.11 presumably

Reply:
+1

Reply:
Good catch. Updated.